aws/requests.yaml

```diff
@@ -1,4 +1,8 @@
 releases:
+    - name: "> 17.0.0"
```

Comment:
worth doing the same for > 16.1.1 as well?

Reply:
If you think so. I'm not sure how previous versions are handled.